crates/nanocld_client/src/cargo.rs

```diff
@@ -84,11 +85,12 @@ impl NanocldClient {
     &self,
     name: &str,
     namespace: Option<String>,
+    force_delete: bool
```

Comment:
We should refactor the function to directly accept the structure, it will be more convenient if we add more options in the future